extension/src/setup/index.ts

```diff
@@ -43,6 +43,7 @@ import { EventName } from '../telemetry/constants'
 import { WorkspaceScale } from '../telemetry/collect'
 import { gitPath } from '../cli/git/constants'
 import { DOT_DVC } from '../cli/dvc/constants'
+import { Toast } from '../vscode/toast'
```

Comment:
There have been mentions about the vscode notifications being easily ignored. Is there a better way to notify the user?

Reply:
Maybe simply add another step in the setup that stays there until the user has commited.